src/model/user.rs

```diff
@@ -190,13 +190,7 @@ impl CurrentUser {
     pub async fn guilds(&self, http: impl AsRef<Http>) -> Result<Vec<GuildInfo>> {
         let mut guilds = Vec::new();
         loop {
-            let mut pagination = http
-                .as_ref()
-                .get_guilds(
-                    &GuildPagination::After(guilds.last().map_or(GuildId(1), |g: &GuildInfo| g.id)),
-                    100,
-                )
-                .await?;
+            let mut pagination = http.as_ref().get_guilds(None, None).await?;
             let len = pagination.len();
             guilds.append(&mut pagination);
             if len != 100 {
```

Comment:
This will break as None defaults to 200, vs the old hardcoded 100. If it gets 200 in response, it will break and not ask for the next page.

Reply:
Oh yeah sorry I see now, should've paid more attention to what I was doing. I'll change it back to `Some(100)` real quick.